File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
#include<unordered_map>
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int,int> numies;
        priority_queue<pair<int,int>> pq;
        vector<int> result;
        
     
        for (int num : nums) {
            numies[num]++;
        }

         for (auto &p : numies) {
            pq.push({p.second, p.first}); 
            
        }
        
          for (int i = 0; i < k; i++) {
            result.push_back(pq.top().second); 
            pq.pop();
        }

        return result;
    }
};

File: Data Structures & Algorithms/validate-parentheses/submission-0.py
class Solution:
    def isValid(self, s: str) -> bool:
        st = []

        for c in range(len(s)):

            if len(st) == 0:
                st.append(s[c])

            elif (
                s[c] == '}' and st[-1] == '{' or
                s[c] == ')' and st[-1] == '(' or
                s[c] == ']' and st[-1] == '['
            ):
                st.pop()

            else:
                st.append(s[c])

        return len(st) == 0